ansible/roles/edu-connector/tasks/configure.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: check if composer.json exists
+  stat:
+    path: "{{ edu_connector_base_dir }}/composer.json"
+  register: composer_json_exists
+  tags:
+    - educonnector
```

Comment:
all the `educonnector` tags are already set in the edu-connector-role, because you set it in the system.yml. Maybe removing them in the tasks for more compact and readable files?

Reply:
I have removed all the tags for eduConnector